expo/gbe/views/create_volunteer_view.py

```diff
@@ -77,3 +70,13 @@ def CreateVolunteerView(request):
                        'page_title': page_title,
                        'view_title': view_title,
                        'nodraft': 'Submit'})
+
+
+def notify_volunteer_reviewers(user_profile):
```

Comment:
If we're going to abstract this, could we bring it higher than the volunteer view level?  Let the view give template name, bidder name, bid type and URL and then use a common function.  Long term, I can see the value to other reviewers getting similar mail for the other bid types, so while we're refactoring, let's make the abstraction level higher.